extension/src/ALObject.ts

```diff
@@ -200,6 +200,15 @@ export class ALObject {
                         newToken.Name = 'RequestOptionsPage';
                         newToken.level = indentation;
                         break;
+                    case 'area':
```

Comment:
Maybe it's time token gets it's own class? Not saying it should be done now but when opportunity rises :)

Reply:
A LOT of things would need classes here, but not today 😃